tests/test_functions.py

```diff
@@ -63,28 +63,14 @@ class TestBasicFunctions:
     Test for some of the basic functions used in tsdate
     """
 
-    def test_alpha_prob(self):
```

Comment:
Is it worth keeping these tests in a separate `test_utilities` file?

Reply:
I lean towards keeping them where they are, because they're so simple (and at some point, there's diminishing returns from writing microscopic unit tests). But if you'd rather move them, that's fine too!

Reply:
Sure, we could simply dump them in a class at the end of that file, right?